db/migrate/20190211105656_add_questionnaire_to_existing_meetings.decidim_meetings.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+# This migration comes from decidim_meetings (originally 20181107175558)
+
+class AddQuestionnaireToExistingMeetings < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
This migration is failing because there are some Meetings whithout title. Did you find this error?

Reply:
To me, a meeting without title is kind of a bug since the meeting model has a title validation, but I don't know...

Reply:
We need to check in production, if there are some meeting without title.